bosh/opsfiles/rules.yml

```diff
@@ -418,5 +418,5 @@
         service: ALB
         severity: critical
       annotations:
-        summary: '{{$labels.certificate_name}}, expires in less than 15 days: {{$labels.value}}'
-        description: Find out why the certificate is not expiring. The service instance ID is probably in the certificate name
+        summary: '{{$labels.certificate_name}}, expires in less than 15 days. Days until expiration: {{$labels.value}}'
```

Comment:
Will this show a float or integer value? Does it really matter?

Reply:
it's a float. I _could_ try to figure out how to parse it into days:hours:minutes or something, but that sounds like more hassle than it's worth